.circleci/config.yml

```diff
@@ -333,8 +333,8 @@ jobs:
             tar -xf master.tar.gz
             cd emsdk-master
             ./emsdk --notty update-tags
-            ./emsdk --notty install latest-upstream
-            ./emsdk --notty activate latest-upstream
```

Comment:
Do we want to continue to support "latest-upstream" as well as "tot-upstream"?  Presumably emsdk should phase out one of them as IIUC they are basically redundant?

Reply:
Yeah, I think the future is to remove the `latest-upstream` which is from the waterfall, so all we use is from emscripten-releases. For now, there might be other people using it even after this PR lands, so maybe we can keep it around for a bit.

In the long term, after we get rid of the mozilla builder integration, we should have only `latest-upstream, tot-upstream`, and fastcomp versions, and also the ability to get a specific non-latest version.